install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -3,9 +3,8 @@
 #
 enabled: false
 replicaCount: 1
-image:
-  repository: grafana/grafana
-  tag: 6.1.6
+hub: docker.io/grafana
```

Comment:
Sorry but why are we changing helm values.yaml here?  It can introduce a backwards compatible issue.

Would it be sufficient that I just change repository to docker.io:grafana/frana

Reply:
@linsun I am following the direction given to me by @sdake to basically follow the same pattern as we've done with [Prometheus](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/prometheus/values.yaml#L6).

IIUC, you could simply change the repository to `docker.io:grafana/grafana`, though this does not follow the same pattern we've established for other third-party add-on components.

Reply:
i would prefer to keep the old syntax given this installer will be replaced so we don't need to document this change in the upgrade notice.  @sdake thoughts?

BTW @ibm-jason we will need this PR in the new istio/installer repo.

Reply:
@linsun Just to make sure that I'm clear- we want to keep the old syntax, but make sure that the `repository` value is fully qualified in the new installer repo. Correct?

Reply:
Yes, for this PR, I would prefer to keep the old helm values API and add fully qualified image name.  Then if you need this change in the new installer, you need a different PR for the installer repo.